homestead.md

```diff
@@ -192,6 +193,19 @@ If you prefer to use MariaDB instead of MySQL, you may add the `mariadb` option
     provider: virtualbox
     mariadb: true
 
+<a name="aliases"></a>
+### Aliases
+
+One you [ssh into your Homestead machine](#connecting-via-ssh) you have access to many Bash aliases that have been pre-configured. Aliases are like shortcuts that you can configure in order to avoid typing long commands. For example, if you usually type the `clear` command you could configure a `c` alias like this:
```

Comment:
Typo: <s>One</s> Once

Reply:
In this context "once" takes the meaning of the conjunction "as soon as" or "whenever" so I think "once" is correct here.